src/serina.ts

```diff
@@ -6,7 +6,7 @@ import Year from 'filters/year/year';
 import Time from 'filters/time/time';
 import Dates from 'filters/dates/dates';
 import PartialDates from './filters/dates/partialDates';
-import DateAndTime from './filters/dates/dateAndTime';
+import DateAndTime from './filters/dates/dateAndTime/dateAndTime';
```

Comment:
Can we move it out of dates? See `WeekdayAndTime`

Reply:
I think it makes sense to have it there since it is kind of a subset of dates.

Reply:
It doesn't because it is a subset of `date` and `time`, not just `date`